js/data.js

```diff
@@ -12,20 +12,20 @@ const discussedButton = filterForm.querySelector('#filter-discussed');
 
 const bootstrap = async () => {
   try {
-    const pictures = await loadPictures();
-    return pictures;
+    const photos = await loadPhotos();
+    return photos;
   } catch (error) {
     showErrorMessage();
   }
 };
 
-const photos = await bootstrap();
-let filteredPhotos = initGalleryFilters(photos, 'default');
+const allPhotos = await bootstrap();
+let filteredPhotos = initGalleryFilters(allPhotos, 'default');
 
 const rePaintDefault = () => {
-  const pictures = document.querySelectorAll('.picture');
-  pictures.forEach((item) => item.remove());
-  const elements = initGalleryFilters(photos, 'default');
+  const photos = document.querySelectorAll('.picture');
```

Comment:
У тебя и массив из ответа сервера это photos и массив нод тоже. Иногда одного или даже слов мало для названия
```suggestion
  const photoNodes= document.querySelectorAll('.picture');
```